src/careamics/prediction_utils/lvae_prediction.py

```diff
@@ -90,6 +150,7 @@ def lvae_predict_mmse(
         # store sample predictions
         sample_predictions[mmse_idx, ...] = sample_prediction
 
-    # take the mean of the sample predictions
     mmse_prediction = torch.mean(sample_predictions, dim=0)
```

Comment:
We will also need pixelwise variance for calibration estimation. `torch.std(sample_predictions, dim=0)` is needed along with mmse_prediction.

Reply:
Ah I see, ok, I will find a way to add it in

Reply:
@ashesh-0 so the pixel-wise standard deviation is only needed for calibration? and will the standard deviation tiles also need to be stitched together?

Reply:
Currently I work with stiched std and stiched prediction. however, that is not necessary. but what is necessary is that we remove the correct overlap from the tiles. so, in short,  if it is not that much pain, then tiled std would be better but i can work with tiles. 
and yes, pixel-wise std is needed just for calibration. but then it is a property of mmse. so whenever we output mmse prediction, we should also output mmse std.

Reply:
ok makes sense, thanks